.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+
+sudo: false
+
+node_js:
+  - 'stable'
```

Comment:
Does `'stable'` make sense? VS Code bundles its own Electron/Node.js/v8, correct? Should that be used instead?

Also, is there any value in asking Travis CI to build for OS X? Probably not that much I suppose?

Reply:
@EricJizbaMSFT You have any thoughts on these?  

@rcjsuen It's what we've been using for our other extensions without issue, I'm going to go ahead and merge for now, but we can always revisit.